README.rst

```diff
@@ -43,6 +53,17 @@ How can I run the tests?
    will reduce the danger of race conditions when building containers)
 4. Run ``tox -e $language_stack -- -n auto``
 
+Environment variables
+^^^^^^^^^^^^^^^^^^^^^
+
+One can use the following environment variables to configure the behaviour of the BCI-Tests
+
+.. code-block:: shell-session
+
+    # export OS_VERSION=15.6                         # Target SLES version
+    # export CONTAINER_RUNTIME=podman                # Defaults to podman
+    # export TARGET=ibs-cr                           # ibs-cr = ToTest, ibs = SUSE:SLE-15-SP*:Update:BCI, obs = devel:BCI:*, factory-totest = Factory
```

Comment:
I think this is great, but we should have a section for each environment variable explaining what it means? happy to help with this myself.

Reply:
I've added a separate block now, please check again if that matches what you meant.